home.html

```diff
@@ -20,41 +20,72 @@ <h1>Good Morning, Alex.</h1>
 
     <main class="main">
       <section class="newsfeed">
-          <ul class="newsfeed__list">
-            <li>
-              <div class="content-container">
-                <img class="content-container__image" src="https://source.unsplash.com/200x280?nature" alt="newsfeed image">
-                <div class="content-container__wrapper">
-                  <img class="content-container__wrapper--avatar" src="https://source.unsplash.com/100x100/?men" alt="newsfeed user">
-                </div>
+        <ul class="newsfeed__list">
+          <li>
+            <div class="content-container">
+              <img
+                class="content-container__image"
+                src="https://source.unsplash.com/200x280?nature"
+                alt="newsfeed image"
+              />
+              <div class="content-container__wrapper">
+                <img
+                  class="content-container__wrapper--avatar"
+                  src="https://source.unsplash.com/100x100/?men"
+                  alt="newsfeed user"
```

Comment:
I would suggest an empty alt attribute here as this does not provide any relevant information.